docs/running-calyx/fud2/rhai-api.md

```diff
@@ -0,0 +1,152 @@
+# Rhai API
+
+## High Level Rhai
```

Comment:
How about including one self-contained (complete) Rhai script to show how everything fits together? Then you can read the subsections to learn more about the individual constructs you can use.